main_ovs.py

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env python
+import nltk
+import re
+import string
+import json
+
+
+class FileReader:
+    @staticmethod
+    def read_file(name):
+        """ func to read from file """
+        with open(name) as fl:
+            return fl.readlines()
+
+
+class JsonWriter:
+    @staticmethod
+    def write_json(name, jsn):
+        with open(name, 'w') as file:                   # serialization
```

Comment:
This class is very good in terms of [SRP](https://en.wikipedia.org/wiki/Single_responsibility_principle), but this comment is redundant - the code is pretty self-documenting.